src/elexmodel/handlers/s3.py

```diff
@@ -128,7 +132,7 @@ def wait_for_versions(self, q):
             try:
                 future.result()
                 yield version, data
-            except Exception as e:
+            except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
Hmmmmmm...I think you're right.  Let me see what happens if I switch all of these to `noqa` 🤔

Reply:
Oh wow, so TIL that `pre-commit` looks at `noqa` lines which are part of the PEP8 standard, but `pylint` doesn't look at `noqa` at all 🤔  https://github.com/pylint-dev/pylint/issues/2493

So I guess I'll leave these `pylint` lines in so that `pylint` doesn't issue these complaints when we run `tox` 🤔

Reply:
We didn't really deal with the issues with tox in the past, and I don't think we should moving forward. I think we can stick to just ones that make our pre-commit hooks fail.